ingress-nginx-controller-1.12.yaml

```diff
@@ -498,6 +502,73 @@ pipeline:
         && setcap -v cap_net_bind_service=+ep ${{targets.destdir}}/usr/bin/nginx
 
 subpackages:
+  - name: ingress-nginx-controller-bitnami-compat-${{vars.nginx-ingress-major-minor}}
+    description: Compatibility package with bitnami/nginx-ingress-controller image
+    dependencies:
+      provides:
+        - ingress-nginx-controller-bitnami-compat=${{package.full-version}}
+      runtime:
+        - bash # https://github.com/bitnami/charts/blob/db53e622ecf1be5a78d1ce683cec8baa41110fa3/bitnami/nginx-ingress-controller/templates/controller-deployment.yaml#L89
+        - coreutils
+        - curl
+        - findutils
+        - gmp
+        - gnutls
+        - grep
+        - ingress-nginx-controller-compat-${{vars.nginx-ingress-major-minor}}
```

Comment:
Please add some comment on why this is added.

I think we're using compat because it also ships symlinks we depend on, but having bitnami-compat use compat is not something we do often.

Let's keep the package, just add a comment.

Reply:
Thanks, I agree. Done, PTAL @wojciechka